src/data-interfaces/HoloFuelDnaInterface.js

```diff
@@ -197,39 +197,31 @@ function presentTransaction (transaction) {
 const HoloFuelDnaInterface = {
   user: {
     get: async () => {
-      const myProfile = await createZomeCall('profile/get_my_profile')()
-      if (myProfile.Err) throw new Error(`There was an error locating the current holofuel agent profile. ERROR: ${myProfile.Err}.`)
+      const { agent_address: id, avatar_url: avatarUrl, nickname, Err } = await createZomeCall('profile/get_my_profile')()
+      if (Err) throw new Error(`There was an locating your holofuel agent ID. ERROR: ${Err}.`)
       return {
-        id: myProfile.agent_address,
-        avatarUrl: myProfile.avatar_url,
-        nickname: myProfile.nickname
+        id,
+        avatarUrl: avatarUrl,
+        nickname: nickname
       }
     },
     getCounterparty: async ({ agentId }) => {
       const counterpartyProfileArray = await createZomeCall('profile/get_profile')({ agent_address: agentId })
-      if (counterpartyProfileArray === 'Err' || !counterpartyProfileArray[0]) {
-        return {
-          id: agentId,
-          avatarUrl: '',
-          nickname: '',
-          notFound: true
-        }
-      }
+      if (counterpartyProfileArray.Err) throw new Error(`There was an locating the holofuel agent with ID: ${agentId}. ERROR: ${counterpartyProfileArray.Err}. `)
```

Comment:
missed the word 'error' in this message.

Also, should this variable be called "array". Seems like it's just a `counterpartyProfile`

Reply:
Also, it seems like this shouldn't throw the error. We're not catching it anywhere? So I think we should "catch" it here, and just return 
```
{
  id: null,
  avatarUrl: null,
  nickname: null
}
```
when `Err` is set

Reply:
I also prefer to maintain the error catching/handling in the dna logic, and I also think we should return an error to the console to demarcate undesired/unexpected behavior.  Therefore, I'll return the null values as well as add a console.error.